std/container/rbtree.d

```diff
@@ -690,7 +690,7 @@ private struct RBRange(N)
      *
      * Complexity: amortized $(BIGOH 1)
      */
-    void popFront()
+    void popFront() @safe
```

Comment:
`@safe` should be inferred, because this is in a template. If it's not inferred, that's likely a compiler bug.

Reply:
That's what I was afraid of... Getting reduced case turns to be non-trivial.